xbmc/games/GameUtils.cpp

```diff
@@ -252,3 +282,34 @@ bool CGameUtils::IsStandaloneGame(const ADDON::AddonPtr& addon)
 
   return false;
 }
+
+bool CGameUtils::Install(const std::string& gameClient)
+{
+  // If the addon isn't installed we need to install it
+  bool bInstalled = CServiceBroker::GetAddonMgr().IsAddonInstalled(gameClient);
+  if (!bInstalled)
+  {
+    ADDON::AddonPtr installedAddon;
+    bInstalled = ADDON::CAddonInstaller::GetInstance().InstallModal(
+        gameClient, installedAddon, ADDON::InstallModalPrompt::NO_PROMPT);
+    if (!bInstalled)
+    {
+      CLog::Log(LOGERROR, "Game utils: Failed to install %s", gameClient.c_str());
+      // "Error"
+      // "Failed to install add-on."
+      MESSAGING::HELPERS::ShowOKDialogText(257, 35256);
```

Comment:
Not sure if this is actually needed but it seems to be what we do everywhere else.
```suggestion
      MESSAGING::HELPERS::ShowOKDialogText(CVariant{257}, CVariant{35256});
```